internal/namespaces/ssh-key/ssh_key.go

```diff
@@ -0,0 +1,128 @@
+package sshkey
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"path"
+	"reflect"
+
+	"github.com/scaleway/scaleway-cli/internal/args"
+	"github.com/scaleway/scaleway-cli/internal/core"
+	"github.com/scaleway/scaleway-cli/internal/interactive"
+	account "github.com/scaleway/scaleway-sdk-go/api/account/v2alpha1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+func GetCommands() *core.Commands {
+	return core.NewCommands(
+		sshKeyCommand(),
+		initCommand(),
+	)
+}
+
+func sshKeyCommand() *core.Command {
+	return &core.Command{
+		Short:     `Handle SHH key`,
+		Long:      `Handle SHH key.`,
+		Namespace: "ssh-key",
+		NoClient:  true,
+		ArgsType:  reflect.TypeOf(args.RawArgs{}),
+	}
+}
+
+type InitArgs struct {
+	AddSHHKey *bool
+}
+
+func initCommand() *core.Command {
+	return &core.Command{
+		Short:     `Initiliaze SHH key`,
+		Long:      `Initiliaze SHH key.`,
+		Namespace: "ssh-key",
+		Resource:  "init",
+		ArgsType:  reflect.TypeOf(InitArgs{}),
+		ArgSpecs: core.ArgSpecs{
+			{
+				Name:  "add-ssh-key",
+				Short: "Whether the ssh key for managing instances should be uploaded automatically",
+			},
+		},
+		Run: InitRun,
+	}
+}
+
+func InitRun(ctx context.Context, argsI interface{}) (i interface{}, e error) {
+	args := argsI.(*InitArgs)
+
+	// Explanation
+	_, _ = interactive.Println("An SSH key is required if you want to connect to a server. More info at https://www.scaleway.com/en/docs/configure-new-ssh-key/")
+
+	// Get default SSH key locally
+	relativePath := ".ssh/id_rsa.pub"
+	filename := path.Join(os.Getenv("HOME"), relativePath)
+	shortenedFilename := "~/" + relativePath
+	localSHHKeyContent, err := ioutil.ReadFile(filename)
+
+	addKeyInstructions := `scw account ssh-key add name=my-key key="($)(cat path/to/my/key.pub)"`
+
+	// Early exit if key is not present locally
+	if os.IsNotExist(err) {
+		return "We did not find an ssh key at " + shortenedFilename + "\n" +
+			"You can add one later using:\n" +
+			"  " + addKeyInstructions, nil
+	} else if err != nil {
+		return nil, err
+	}
+
+	// Get all SSH keys from Scaleway
+	client := core.ExtractClient(ctx)
```

Comment:
Should we add a `core.ExtractClient()` create a client if not already created ?  This would avoid making the create client method public. And another solution could be `core.ExtractClientOrCreate()` that could return error if needed.

Reply:
what's the problem with making `core.ExtractClient()` public ?

Reply:
let's do 

> add a core.ExtractClient() create a client if not already created ? This would avoid making the create client method public.

Reply:
the second solution actually has less impact
It might be a better approach